napari/_vispy/vispy_scale_bar_visual.py

```diff
@@ -24,21 +28,22 @@ def __init__(self, viewer, parent=None, order=0):
             ]
         )
         self._default_color = np.array([1, 0, 1, 1])
-        self._target_length = 100
+        self._target_length = 150
         self._scale = 1
+        self._quantity = UNIT_REG("")  # unit-less
 
         self.node = Line(
             connect='segments', method='gl', parent=parent, width=3
         )
         self.node.order = order
         self.node.transform = STTransform()
 
-        self.text_node = Text(pos=[0, 0], parent=parent)
+        self.text_node = Text(pos=[0.5, -1], parent=self.node)
```

Comment:
Just curious why does the parent need to change here? Is it so the text now always appears centered and below the line? Maybe can you add that as a comment

Reply:
Yes, that's correct. I've added a comment to indicate this.